app/models/remove.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Remove < ApplicationRecord
```

Comment:
I think it'd be clearer to have a more noun-y name here, as that's how we do most of the models. Had a chat with @kevindew and we came up with 'Retirement' and 'Removal'. What do you think?

Reply:
I've renamed the models in a new branch: https://github.com/alphagov/content-publisher/pull/575